AutomatedTesting/Gem/PythonTests/scripting/scripting_utils/script_canvas_component.py

```diff
@@ -0,0 +1,133 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+We are deprecating this file and moving the constants to area specific files. See the EditorPythonTestTools > consts
+folder for more info
+"""
+from editor_python_test_tools.editor_entity_utils import EditorEntity
+import azlmbr.scriptcanvas as scriptcanvas
+from enum import Enum
+import azlmbr.math as math
+from consts.scripting import (SCRIPT_CANVAS_UI)
+
+import azlmbr.bus as bus
+import azlmbr.editor as editor
+import azlmbr.entity as entity
+import azlmbr.legacy.general as general
+import azlmbr.object
+
+# component property path
+SCRIPT_CANVAS_COMPONENT_PROPERTY_PATH = "Configuration|Source"
+
+# path parts
+PROPERTIES = "Configuration|Properties|"
+UNUSED_VARIABLES = "Unused Variables|"
+VARIABLES = "Variables|"
+DATUM_PATH = "Datum|Datum|value|"
+
+
+class VariableState(Enum):
+    UNUSEDVARIABLE = 0
+    VARIABLE = 1
+
+
+class ScriptCanvasComponent:
+
+    def __init__(self):
+        self.editor_entity = None
+        self.sc_component = None
```

Comment:
Let's not use shorthand and practice bad conventions that started due to limited memory space.

Refactor to `self. script_canvas_componenet`